packages/trpc/server/routers/loggedInViewer/locationOptions.handler.ts

```diff
@@ -4,22 +4,30 @@ import { getTranslation } from "@calcom/lib/server/i18n";
 import { prisma } from "@calcom/prisma";
 import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
 
+import type { TLocationOptionsInputSchema } from "./locationOptions.schema";
+
 type LocationOptionsOptions = {
   ctx: {
     user: NonNullable<TrpcSessionUser>;
   };
+  input: TLocationOptionsInputSchema;
 };
 
-export const locationOptionsHandler = async ({ ctx }: LocationOptionsOptions) => {
+export const locationOptionsHandler = async ({ ctx, input }: LocationOptionsOptions) => {
   const credentials = await prisma.credential.findMany({
     where: {
-      userId: ctx.user.id,
+      OR: [{ userId: ctx.user.id }, input?.teamId ? { teamId: input.teamId } : {}],
```

Comment:
Here the same as above, adjust whereConditional if teamId is provided.

Reply:
We're not including locations for V1 but will keep this in mind.